### PR TITLE
Automated Test - COMMERCE-5706 | master

### PR DESCRIPTION
**Pull request by wesleyroberts**

### CODE REVIEW

**Comment by wesleyroberts** [comment]
https://issues.liferay.com/browse/COMMERCE-5706 ; cc: @riccardo-alberti

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path [hunk 1]
```diff
@@ -550,6 +550,11 @@
 	<td>//div[contains(@class,'side-panel-iframe-header')]//button[contains(@class,'side-panel-iframe-close')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHECK_WAREHOUSE_COLLAPSE_OPEN</td>
+	<td>//a[contains(.,'${specifiedCollapse}')]/../../..//div[contains(@class,'show')]</td>
```
Comment (iovaneale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path:555]: Please use the following standard when putting vars into locators: ${key_varName}

e.g. ${key_specifiedCollapse}

**Review by iovaneale — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		if (!(IsElementPresent(
+					specifiedCollapse = "Geolocation",
+					locator1="CommerceEntry#CHECK_WAREHOUSE_COLLAPSE_OPEN"))) {
+			Click(
+				key_contentType = "Geolocation",
+				locator1 = "Notifications#NOTIFICATION_DELIVERY_PANEL");
+
+			Pause(locator1 = "1000");
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro]: changes made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro [hunk 3]
```diff
@@ -382,6 +426,77 @@ definition {
...
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		if (!(IsElementPresent(
+					specifiedCollapse = "Address",
+					locator1="CommerceEntry#CHECK_WAREHOUSE_COLLAPSE_OPEN"))) {
+			Click(
+				key_contentType = "Address",
+				locator1 = "Notifications#NOTIFICATION_DELIVERY_PANEL");
+
+			Pause(locator1 = "1000");
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro]: changes made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path (hunk 1 above)
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path]: changes made

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
@wesleyroberts 

This Pull Request has conflicting files. Please fix and resend.

**Comment by wesleyroberts** [comment]
i wiil change an resend.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by wesleyroberts** [comment]
I rebase the branch

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:commerce-functional

**Comment by nikki-pru** [comment]
Combined: https://github.com/liferay-commerce/liferay-portal/pull/1255
Forwarded: https://github.com/brianchandotcom/liferay-portal/pull/102208

**Pull request closed without merging**